packages/core/scss/components/dock-manager/_layout.scss

```diff
@@ -196,6 +197,7 @@
         pointer-events: all;
         outline-width: $kendo-dock-indicator-outline-width;
         outline-style: $kendo-dock-indicator-outline-style;
+        z-index: k-z-index("top", 1);
```

Comment:
Do we need such a high high z-index here? Since all _k-dock-indicator_ elements are already wrapped inside the k-_dock-navigator_ container, I believe this rule may not be necessary. The existing structure should already ensure the correct stacking context.